SConstruct

```diff
@@ -42,10 +49,11 @@ vars.Add(BoolVariable('debug', 'Set to build debug target', 0))
 vars.Add(BoolVariable('trace', 'Set to compile trace debug statements', 0))
 vars.Add(BoolVariable('force_zero_revision', 'Force Mercurial revision to zero to prevent rebuilds after committing when developing', 0))
 vars.Add(BoolVariable('build_java', 'Set to build the Kalisko java source tree', 0))
-vars.Add(ListVariable('modules', 'Attempts to build the selected modules', [], modules))
-vars.Add(ListVariable('test_modules', 'Attempts to build the selected unit test modules', [], test_modules))
-vars.Add(ListVariable('exclude', 'Prevents the selected modules from being built', [], modules))
-vars.Add(ListVariable('test_exclude', 'Prevents the selected unit test modules from being built', [], test_modules))
+vars.Add(BoolVariable('expand_module_deps', 'Set to build all runtime dependencies of the passed modules', 0))
```

Comment:
We should set this to default enabled since there seem to be more cases where it's useful than where it's not.